### PR TITLE
Add Support for Custom Chart Values to Master Helmfile

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added `uuidgen`
* Added custom tabs for portal
* Added custom grafana dashboards

## Why
* To make documentation examples in docs working in the geodesic shell
* To extend tabs on the geodesic module level
* To extend grafana dashboards on the geodesic module level

### CODE REVIEW

File: rootfs/conf/kops/helmfile.yaml [hunk 1]
```diff
@@ -675,6 +675,8 @@ releases:
     default: "true"
   chart: "coreos-stable/kube-prometheus"
   version: "0.0.66"
+  values:
+    - "custom_values/kube-prometheus.grafana.dashboards.yaml"
```
Comment (osterman) [inline rootfs/conf/kops/helmfile.yaml]: I like this solution, but I don't like the introduction of a new `custom_values` namespace. Can we just stick with `values/`?

File: rootfs/conf/kops/helmfile.yaml [hunk 2]
```diff
@@ -1097,33 +1099,34 @@ releases:
...
-      value: '{{ env "PORTAL_BACKEND_ALERTS_ENDPOINT" }}'
+      value: '{{ coalesce (env "PORTAL_BACKEND_ALERTS_ENDPOINT") "http://kube-prometheus-alertmanager.monitoring:9093" }}'
     - name: "backends.alerts.external"
       value: "false"
 
     - name: "backends.prometheus.name"
       value: '{{ coalesce (env "PORTAL_BACKEND_PROMETHEUS_NAME") "Prometheus" }}'
     - name: "backends.prometheus.endpoint"
-      value: '{{ env "PORTAL_BACKEND_PROMETHEUS_ENDPOINT" }}'
+      value: '{{ coalesce (env "PORTAL_BACKEND_PROMETHEUS_ENDPOINT") "http://kube-prometheus-prometheus.monitoring:9090" }}'
```
Comment (osterman) [inline rootfs/conf/kops/helmfile.yaml]: Btw, I learned it's possible to write:
```
env "PORTAL_BRAND_IMAGE_URL" | default "http://kube-prometheus-prometheus.monitoring:9090"
```

File: rootfs/conf/kops/helmfile.yaml [hunk 3]
```diff
@@ -735,14 +735,14 @@ releases:
 
     ### Optional: KUBE_PROMETHEUS_ALERT_MANAGER_REPLICA_COUNT; e.g. 4
     - name: "alertmanager.replicaCount"
-      value: '{{ coalesce (env "KUBE_PROMETHEUS_ALERT_MANAGER_REPLICA_COUNT") "4" }}'
+      value: '{{ env "KUBE_PROMETHEUS_ALERT_MANAGER_REPLICA_COUNT" | "4" }}'
```
[the remark this answers is not in the document]
Comment (goruha) [reply rootfs/conf/kops/helmfile.yaml]: https://github.com/cloudposse/geodesic/pull/137#discussion_r191597359

[the remark this answers is not in the document]
Comment (osterman) [reply rootfs/conf/kops/helmfile.yaml]: I am not able to view link

[the remark this answers is not in the document]
Comment (goruha) [reply rootfs/conf/kops/helmfile.yaml]: @aknysh right. @osterman  pattern do not work

File: rootfs/conf/kops/helmfile.yaml (hunk 2 above)
Comment (goruha) [reply rootfs/conf/kops/helmfile.yaml]: Fixed

File: rootfs/conf/kops/helmfile.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (goruha) [reply rootfs/conf/kops/helmfile.yaml]: sorry, I missed `default`. Fixed

**Pull request merged**